assets/templates/partials/release/contents/about-the-data.tmpl

```diff
@@ -1,11 +1,21 @@
 <div class="about-the-data">
   {{ if .Description.Census2021 }}
     <div class="ons-u-mb-l census-2021">
-      <img
-        src="https://cdn.ons.gov.uk/assets/images/census-logo/logo-census-2021-purple-landscape.svg"
-        alt="Census 2021 logo"
-        class="ons-u-mb-xs"
-      >
+      {{ if eq .Language "en" }}    
```

Comment:
The default language is English so this should be the fallback option in the `else` block, and this guard should test for `cy`.

Reply:
And remove the trailing spaces

Reply:
This is done now.